{"CVE Code": "CVE-2019-10742", "Description": "Axios up to and including 0.18.0 allows attackers to cause a denial of service (application crash) by continuing to accepting content after maxContentLength is exceeded.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2019-10742", "Vulnerability Categories": "Denial of service", "Affected Products": "\nAxios\u00bbAxios\u00bbForNode.jsVersions  up to, including, (<=) 0.18.0cpe:2.3:a:axios:axios:*:*:*:*:*:node.js:*:*Matching versions", "EPSS Score": "0.36% Probability of exploitation activity in the next 30 days\n~ 72 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2024-12-17  0.33%  0.36%  +0.03 \n 2  2024-09-27  0.28%  0.33%  +0.05 \n 3  2024-04-30  0.20%  0.28%  +0.08 \n 4  2023-03-07  5.74%  0.20%  -5.53 ", "CVSS Scores": "5.0", "CWE": [{"CWE_ID": "CWE-755 Improper Handling of Exceptional Conditions", "CWE_link": "/cwe-details/755/Improper-Handling-of-Exceptional-Conditions.html"}], "Reference": [{"ref_link": "https://app.snyk.io/vuln/SNYK-JS-AXIOS-174505", "ref_desc": "Denial of Service (DoS) in axios | SnykExploit;Patch;Third Party Advisory", "ref_cve_link": "N/A"}, {"ref_link": "https://github.com/axios/axios/issues/1098", "ref_desc": "Download continues after maxContentLength exceeded \u00b7 Issue #1098 \u00b7 axios/axios \u00b7 GitHubExploit;Third Party Advisory", "ref_cve_link": "N/A"}, {"ref_link": "https://github.com/axios/axios/pull/1485", "ref_desc": "Destroy stream on exceeding maxContentLength (fixes #1098) by resure \u00b7 Pull Request #1485 \u00b7 axios/axios \u00b7 GitHubPatch;Third Party Advisory", "ref_cve_link": "N/A"}]}